exporter/googlemanagedprometheusexporter/README.md

```diff
@@ -63,16 +63,6 @@ receivers:
                 - action: labelmap
                 regex: __meta_kubernetes_pod_label_(.+)
 processors:
-    # groupbyattrs promotes labels from metrics to resources, allowing them to
-    # be added to the prometheus_target monitored resource.
-    # This allows exporters which monitor multiple namespaces, such as
-    # kube-state-metrics, to override the namespace in the resource by setting
-    # metric labels.
-    groupbyattrs:
-      keys:
-        - namespace
```

Comment:
Lee recommended that we copy the GMP logic of prepending `exported_` to all labels that would otherwise collide with prometheus_target labels.  We should add a processor that renames metric labels that can collide (job, instance, cluster, location, namespace, project_id) to prefixed labels.

Reply:
Updated the example with the transform processor to recommend `exported_` and added a block that handles all of these

Reply:
Is there a way for us to include that behavior in the config up here?  It would be great if that was the "default" behavior.

Reply:
sure, moved it up to the main example